surveys/2024-04/questions.md

```diff
@@ -0,0 +1,142 @@
+# Survey April 2024
+
+## Demographics
+
+- Where are you from? (countries list if possible - or open question)
+    - country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
+
+- In what kind of organizations are you working in?
+    - big tech
+    - company
+    - startup
+    - university
+    - individual
+    - other
+
+- Are you part/following some Working Groups? Are you involved in some Node.js community  Working Groups ? If yes which one (open question)
+
+## Node.js Usage
+
+- Which groups do you identify with? (multiple choice)
+    - Direct end users: Users who run tools themselves
+    - Application operators: Users who interacts with existing running applications
+    - Application developers: Frontend tools consumers, backend server authors, tools authors
+    - Library & package authors: Users who write libraries and packages to be included on other applications
+    - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+    - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+
+- Are you part of a group not covered ? If yes which one ? (open question)
+
+- What is your use cases of Node.js ? (if multiple use case, separate with a comma) (open question)
+
+## What Node.js binaries do you use
+
+- What is your Operating System for your local development environment ? (single choice - Other should give an input)
+	- Linux
+	- Windows
+	- MacOs
+	- Other?
+
+- What is your Operating System in which you are running Node.js in production ? Do you plan to change that in the near future? (single choice - Other should give an input)
+	- Linux/Unix
+	- Windows
+	- MacOs
+	- Other?
+
+- What is your architecture in which you are running Node.js with Node.js ? Do you plan to change that in the near future? (single choice - Other should give an input)
```

Comment:
I think it's not uncommon to use multiple machines to run Node.js, which may have different architecture.
```suggestion
- What is your architecture in which you are running Node.js? Do you plan to change that in the near future? (multiple choice - Other should give an input)
```
Alternatively we could change the question to be limited to the main development machine.
```suggestion
- What architecture is the machine you are running Node.js for development? Do you plan to change that in the near future? (single choice - Other should give an input)
```